pkg/transformer/openshift/openshift_test.go

```diff
@@ -90,5 +95,189 @@ func TestKomposeConvertRoute(t *testing.T) {
 	if route.Spec.Host != sc.ExposeService {
 		t.Errorf("Expected %s for Spec.Host, actual %s", sc.ExposeService, route.Spec.Host)
 	}
+}
+
+//Test getting git remote url for a directory
+func TestGetGitRemote(t *testing.T) {
+	var output string
+	var err error
+
+	gitDir := testutils.CreateLocalGitDirectory(t)
+	testutils.SetGitRemote(t, gitDir, "newremote", "https://git.test.com/somerepo")
+	testutils.CreateGitRemoteBranch(t, gitDir, "newbranch", "newremote")
+	dir := testutils.CreateLocalDirectory(t)
+	defer os.RemoveAll(gitDir)
+	defer os.RemoveAll(dir)
+
+	testCases := map[string]struct {
+		expectError bool
+		dir         string
+		branch      string
+		output      string
+	}{
+		"Get git remote for branch success":   {false, gitDir, "newbranch", "https://git.test.com/somerepo.git"},
+		"Get git remote error in non git dir": {true, dir, "", ""},
+	}
+
+	for name, test := range testCases {
+		t.Log("Test case: ", name)
+		output, err = getGitCurrentRemoteUrl(test.dir)
+
+		if test.expectError {
+			if err == nil {
+				t.Errorf("Expected error, got success instead!")
+			}
+		} else {
+			if err != nil {
+				t.Errorf("Expected success, got error: %v", err)
+			}
+			if output != test.output {
+				t.Errorf("Expected: %#v, got: %#v", test.output, output)
+			}
+		}
+	}
+}
+
+// Test getting current git branch in a directory
+func TestGitGetCurrentBranch(t *testing.T) {
+	var output string
+	var err error
+
+	gitDir := testutils.CreateLocalGitDirectory(t)
+	testutils.SetGitRemote(t, gitDir, "newremote", "https://git.test.com/somerepo")
+	testutils.CreateGitRemoteBranch(t, gitDir, "newbranch", "newremote")
+	dir := testutils.CreateLocalDirectory(t)
+	defer os.RemoveAll(gitDir)
+	defer os.RemoveAll(dir)
+
+	testCases := map[string]struct {
+		expectError bool
+		dir         string
+		output      string
+	}{
+		"Get git current branch success": {false, gitDir, "newbranch"},
+		"Get git current branch error":   {true, dir, ""},
+	}
+
+	for name, test := range testCases {
+		t.Log("Test case: ", name)
+		output, err = getGitCurrentBranch(test.dir)
+
+		if test.expectError {
+			if err == nil {
+				t.Errorf("Expected error, got success instead!")
```

Comment:
Nitpick: if format specifiers are not used, this can become `t.Error` instead of `t.Errorf`